src/scripts.js

```diff
@@ -60,15 +61,20 @@ const loadOptions = () => {
     cookBook.recipes.forEach(recipe => {
       recipe.tags.forEach(tag => {
         if (!allUniqueTags.includes(tag)) {
-          allUniqueTags.push(tag)
+          allUniqueTags.push(tag);
+          // we changed this from options.
+          optionsContainer.innerHTML += `
+          <div><input type="checkbox" name="check" value="${tag}">
+        <label>${tag}</label></div>
+        `;
         }
       });
     });
-    allUniqueTags.forEach(tag => {
```

Comment:
Rendering just the element itself vs that element wrapped in a div is much cleaner, big fan